winterfell/test/testMailingService.js

```diff
@@ -56,29 +57,27 @@ describe('Mailing', function () {
     var service;
     var testUserInstance;
     var testDocumentInstance;
-    var testUserAddressInstance;
     var testDestinationAddressInstance;
 
     before(function (done) {
         server = require('../app');
         service = new MailingService(server.app);
         service.Lob = mockLob;
 
-        User.create(testUser, function (userErr, user) {
+        User.remove({}, function() {
```

Comment:
I think before tests, it's good practice to reset the db as much as needed. This ensures other tests don't screw up what you're testing